src/lib/symmetric.c

```diff
@@ -613,6 +613,18 @@ pgp_cipher_aead_update(pgp_crypt_t *crypt, uint8_t *out, const uint8_t *in, size
     return true;
 }
 
+void
+pgp_cipher_aead_reset(pgp_crypt_t *crypt)
+{
+    uint32_t flags = BOTAN_CIPHER_UPDATE_FLAG_FINAL;
+    uint8_t  tag[PGP_AEAD_EAX_OCB_TAG_LEN] = {0};
+    size_t   in = 0, out = 0;
+    size_t   len = crypt->aead.decrypt ? crypt->aead.taglen : 0;
+    size_t   dlen = crypt->aead.decrypt ? 0 : crypt->aead.taglen;
+
+    botan_cipher_update(crypt->aead.obj, flags, tag, dlen, &out, tag, len, &in);
```

Comment:
Why not to simply use "botan_cipher_clear"

Reply:
It will also clear the key, while it is reused with each next chunk.

Reply:
ok, I wouldn't guess that. Can you please add comment why we do that, as it's not obvious?
I'll check if there is a way to improve botan's FFI api

Reply:
Updated code with comment. Created issue #645 to sum up what we need to improve in botan

Reply:
perfect